setup.py

```diff
@@ -414,6 +414,6 @@
                 'cylp.py.utils', 'cylp.py.mip','cylp.py.QP'],
       cmdclass={'build_ext': build_ext},
       ext_modules=ext_modules,
-      install_requires=['numpy >= 1.5.0', 'scipy >= 0.10.0'],
+      install_requires=['numpy >= 2.0.0rc1', 'scipy >= 0.10.0'],
```

Comment:
I think this may have been a mistake, your project will continue to work with numpy 1.x so you don’t want to replace this line here.

Reply:
OK, yes, I see.